### PR TITLE
Memory check for cscli dashboard setup

### PR DESCRIPTION
**Pull request by LaurenceJJones**
If total memory < 1GB and force-yes is not provided the user can answer Y/n to continue running the Metabase container if the force-yes flag is provided it will just warn the user that Metabase container requires 1-2GB

Please note the current calculation of memory is set to 1GB as per issue, however, in crowdsec documentation it states 1-2GB

EDIT: The free system memory function was acting a bit strange on my VM it listed only 57MB free instead of 570MB so for the first implementation I thought total memory can be an improvement.

I have done some more testing and the free memory is correct for my VM after system allocation. I can update this PR with the check for free system memory check as well, however, I will wait for feedback.

#826

### CODE REVIEW

File: cmd/crowdsec-cli/dashboard.go [hunk 1]
```diff
@@ -304,3 +323,14 @@ func passwordIsValid(password string) bool {
 	return true
 
 }
+
+func checkSystemMemory() (bool, error) {
+	totMem := memory.TotalMemory()
+	if totMem == 0 {
+		return true, errors.New("Unable to get system total memory")
+	}
+	if uint64(math.Pow(2, 30)) >= totMem {
```
Comment (LaurenceJJones) [inline cmd/crowdsec-cli/dashboard.go:332]: Don't know math.Pow is needed here as the value will always resolve to ```1073741824``` so we could swap this out and hard code the value

File: cmd/crowdsec-cli/dashboard.go [hunk 2]
```diff
@@ -109,6 +112,22 @@ cscli dashboard setup -l 0.0.0.0 -p 443 --password <password>
...
 			}
 			var answer bool
+			if valid, err := checkSystemMemory(); err == nil && !valid {
+				if !forceYes {
+					prompt := &survey.Confirm{
+						Message: "Metabase requires 1-2GB of RAM, your system is below this requirement continue ?",
+						Default: true,
+					}
+					if err := survey.AskOne(prompt, &answer); err != nil {
+						log.Fatalf("unable to ask about RAM check: %s", err)
```
Comment (buixor) [inline cmd/crowdsec-cli/dashboard.go:122]: we might want to warn/error on this but not fatal : if for any reason the check errors, we're preventing the user from setting up the dashboard

**Review by buixor — CHANGES_REQUESTED** [review]
minor suggestion, overall lgtm :+1:

**Review by he2ss — APPROVED** [review]
lgtm

**Pull request merged**